torch/optim/adam.py

```diff
@@ -25,8 +53,6 @@ class Adam(Optimizer):
 
     .. _Adam\: A Method for Stochastic Optimization:
         https://arxiv.org/abs/1412.6980
-    .. _Decoupled Weight Decay Regularization:
```

Comment:
This is not true anymore?

Reply:
This paper contains AdamW rather than Adam, that's why i deleted this :) thanks for pointing it out.

Reply:
ok